src/fides/api/schemas/privacy_experience.py

```diff
@@ -56,56 +86,45 @@ def validate_regions(
 
     @root_validator
     def validate_attributes(cls, values: Dict[str, Any]) -> Dict[str, Any]:
-        """Validate minimum set of required fields exist given the type of component and/or delivery_mechanism"""
+        """Validate minimum set of required fields exist given the type of component"""
         component: Optional[ComponentType] = values.get("component")
```

Comment:
Updating the logic here with my understanding of what fields are required when saving an overlay.  Currently overlays and privacy center experience configs are stored in the same table, as I feel there are a lot of overlapping fields. The Privacy Center needs a subset of the Overlay fields.